source/MRMesh/MRGcodeProcessor.cpp

```diff
@@ -72,6 +78,72 @@ GcodeProcessor::MoveAction GcodeProcessor::processLine( const std::string_view&
     return {};
 }
 
+void GcodeProcessor::setRotationParams( RotationParameterName paramName, const Vector3f& rotationAxis )
+{
+    const int intParamName = int( paramName );
+    const bool validParamName = intParamName >= int( RotationParameterName::A ) && intParamName < int( RotationParameterName::Count );
+    assert( validParamName );
+    if ( !validParamName )
+        return;
```

Comment:
looks like excessive checks

Reply:
no, not all RotationParameterName have axis (RotationParameterName::None or Count)